### PR TITLE
Update README.md for SD Fine Tuning

### PR DESCRIPTION
**Pull request by mvpatel2000**

### CODE REVIEW

**Review by A-Jacobson — APPROVED** [review]
Have you tried the pip install for xformers? I trained this with PyTorch 1.13 but the readme says they only support 13. If you've tried it and it works go ahead and merge.

File: examples/stable_diffusion/README.md [hunk 1]
```diff
@@ -40,8 +40,7 @@ cd examples/stable_diffusion
 xformers contains faster, more memory effecient transformer layers. But can take a while to install.
 
 ```
-pip install ninja # Faster xformers install
-pip install git+https://github.com/facebookresearch/xformers.git@3df785ce54114630155621e2be1c2fa5037efa27#egg=xformers
+pip install xformers
```
Comment (dakinggg) [inline examples/stable_diffusion/README.md:43]: Is the install still slow? If not, should we just put it in requirements?

**Comment by mvpatel2000** [comment]
> Have you tried the pip install for xformers? I trained this with PyTorch 1.13 but the readme says they only support 13. If you've tried it and it works go ahead and merge.

Let me verify fine tune gives same performance with 1.13. I was able to get it to work with the full training runs....

**Comment by mvpatel2000** [comment]
Closing in favor of https://github.com/mosaicml/examples/pull/175

**Pull request closed without merging**